engine/src/main/resources/org/pentaho/di/pan/messages/messages_en_US.properties

```diff
@@ -69,4 +69,5 @@ Pan.ComdLine.JarFile=specifies the jar filename
 Pan.Log.LoadingTransXML=Loading transformation from XML file [{0}]
 Pan.Error.TransJVMExitCodeInvalid=The transformation variable {0} is not an integer.  It is set to "{1}".
 Pan.Log.JVMExitCode=Exiting the JVM with exit code {0}.
-Pan.ComdLine.Metrics=Gather metrics during execution
\ No newline at end of file
+Pan.ComdLine.Metrics=Gather metrics during execution
+Pan.Error.ComdLine.RepAndProject=ERROR\: Cannot specify repository and project path at the same time.
```

Comment:
I would still rip this out. We're going to add projects to repos, so this is moot.